lib/ansible/modules/clustering/kubevirt/kubevirt_raw.py

```diff
@@ -237,11 +243,14 @@ def _read_stream(self, resource, watcher, stream):
                     if (not self._use_cdi(annotations, labels) or
```

Comment:
please change `IMPORT_STATUS_KEY` to ```IMPORT_STATUS_KEY = 'cdi.kubevirt.io/storage.pod.phase'```

Reply:
Will do.

Reply:
Btw, what is the rationale for this? When I dump the object returned by the stream on PVC creation, the relevant parts look like this:

```
    "metadata": {
      "annotations": {
        "pv.kubernetes.io/bind-completed": "yes", 
        "pv.kubernetes.io/bound-by-controller": "yes"
      }, 
```

There's no _cdi.kubevirt.io_ of any kind to be found. Is it because we're using different cluster versions or something?

Reply:
That's what's used by the cdi:

https://github.com/kubevirt/containerized-data-importer/blob/9ba4059dfc6657adcf27e2a112273d087d0aa12a/pkg/controller/controller.go#L23

I guess the previous name is just a typo.

Reply:
ok